pages/quiz-page.html

```diff
@@ -0,0 +1,365 @@
+                                        <div
+                                            class="modal-header flex justify-between align-items-center bg-grey-800 text-white p-5">
+                                            <h4 class="h4 text-lg">Quiz Rules</h4>
+                                            <span onclick="closeModal()"
+                                                class="modal-close text-rose-500 text-hover-rose-300">×</span>
+                                        </div>
+                                        <div class="modal-body p-5">
+                                            <p>Some rules in the Modal Body</p>
+                                            <p>Some other rules...</p>
+                                            <p>Some rules in the Modal Body</p>
+                                            <p>Some other rules...</p>
+                                            <p>Some rules in the Modal Body</p>
+                                            <p>Some other rules...</p>
+                                            <p>Some rules in the Modal Body</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="bg-regal-blue-dark rounded shadow-lg h-min">
+                    <div class="h-full  flex flex-col">
+                        <div class="badge-container">
+                            <span
+                                class="absolute font-bold card-badge badge bg-fuchsia-500 text-white bg-hover-fuchsia-300">New
+                                quiz</span>
+                            <div class="p-5">
+                                <img class="img-responsive" src="https://picsum.photos/500" alt="" />
+                            </div>
+                            <button style="background-color: rgba(255, 255, 255, 0.596);" class="btn-close"></button>
+
+                        </div>
+                        <div class="p-5">
+                            <h4 class="h4 font-normal text-lg subtitle1">
+                                <a class="text-black">Some Title</a>
+                            </h4>
+                            <p class="body1 text-sm font-light mb-4 text-black">
+                                Lorem ipsum dolor sit amet, consectetur adipisicing elit. Vero,
+                                possimus nostrum!
+                            </p>
+                            <hr class="mb-3" />
+
+                            <div class="flex justify-between">
+                                <a href="#" class="btn btn-solid-fuchsia text-white">Play Quiz</a>
+
+                                <a onclick="openModal()" id=" modalBtn" class="btn btn-link-fuchsia">Rules</a>
+
+                                <div id="modal" style="z-index: 30;" class="modal">
```

Comment:
The z-index for modals are with a bigger number. Try to optimize that unless you have 29 others in stacking context.

Reply:
yes the was a bog with the z-index so its 30 here